modules/ROOT/pages/query-caches/index.adoc

```diff
@@ -1,48 +1,42 @@
+:description: Different caches to speed up query planning.
+[[query-caches]]
 = Query caches
 
-Cypher utilizes a number of different caches to speed up query planning.
+From Neo4j 5.7 onwards, you can use several different caches to speed up Cypher query planning.
 
-Out of the box, the set of query caches are _per database_.
-That means that for each new database, a new set of caches will be initialized.
+Out of the box, the set of query caches is _per database_.
+That means that a new set of caches is initialized for each new database.
 
-The maximum number of entries per cache is configured with `server.memory.query_cache.per_db_cache_num_entries`.
-
-[NOTE]
-====
-This setting is only deciding cache size when `server.memory.query_cache.sharing_enabled` is set to `false`.
-====
+The maximum number of entries per cache is configured using `server.memory.query_cache.per_db_cache_num_entries`.
+It determines the cache size only when `server.memory.query_cache.sharing_enabled` is set to `false`.
 
 Query caches may consume a lot of memory, especially when running many active databases.
-To combat this, and improve predictability on memory consumption,
-it is possible to configure the DBMS such that only one set of caches is utilized for all databases.
-
-More information on this topic can be found in the following section:
+To tackle this and improve predictability on memory consumption, you can configure the DBMS to use only one set of caches for all databases.
+For more information, see xref:query-caches/unified-query-caches.adoc[Unifying query caches].
 
-* xref:query-caches/unified-query-caches.adoc[]
+[[configure-caches]]
+== Configure caches
 
-[[Configuration]]
-== Configuration summary
-
-See below a summary of query cache configurations.
+The following is a summary of the query cache configurations.
+For more information, see link:{neo4j-docs-base-uri}/operations-manual/{page-version}/reference/configuration-settings/[Operations Manual -> Configuration settings].
 
 .Query cache configurations
-[options="header", width="100%", cols="4a,1a,3a"]
+[options="header", width="100%", cols="4m,3a,1m"]
 |===
-| Setting | Default value | Description
+| Setting
+| Description
+| Default
 
-| `server.memory.query_cache.sharing_enabled`
-| `false`
-| label:enterprise-edition[Enterprise only] Enable sharing cache space between different databases. With this option turned on, databases will share cache space,
-but not cache entries.
+| server.memory.query_cache.sharing_enabled
```

Comment:
I would still find it useful if all these mentions of the exact settings were clickable links to the exact sub-section in the operations manual.